flytekit/types/schema/types_pandas.py

```diff
@@ -147,6 +147,9 @@ def to_python_value(
         r = PandasSchemaReader(local_dir=local_dir, cols=None, fmt=SchemaFormat.PARQUET)
         return r.all()
 
+    def to_html(self, ctx: FlyteContext, python_val: pandas.DataFrame):
+        return python_val.to_html()
```

Comment:
we just want to print the column names by default. Dumping the entire pandas dataframe is not a great idea

Reply:
I think information in DataFrame.info() or DataFrame.describe() is sufficient for the built-in renderer

Reply:
What if we used [typing.Annotated](https://docs.python.org/3/library/typing.html#typing.Annotated) to define what is going to be rendered? By default it only does `df.describe()` but it could potentially run other kinds of aggregation (e.g. list of unique values) or maybe parameters to [`df.describe()`](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.describe.html).